### DISCUSSION TITLE
1.19.3

### DISCUSSION THREAD

**Discussion by JohnnyJosda** [IrisShaders/Iris#1968] [Q&A] [answered]
Where did the version 1.19.3 go?

**Comment by Rhomboid** [comment]
What do you mean? It didn't go anywhere.

https://modrinth.com/mod/iris/versions?g=1.19.3

**Reply by JohnnyJosda** [reply]
Not the 1.5 version, 1.19.3 does not have the 1.6 iris

**Reply by Timolino456** [reply, marked as answer]
Support for 1.19.3 has been dropped, use 1.19.4 instead if you want to use Iris 1.6.
